marimo/_plugins/ui/_impl/table.py

```diff
@@ -725,6 +784,15 @@ def clamp_rows_and_columns(manager: TableManager[Any]) -> JSONType:
             return SearchTableResponse(
                 data=clamp_rows_and_columns(self._manager),
                 total_rows=self._manager.get_num_rows(force=True) or 0,
+                # The __init__ will just call this with an arbitrary offset,
+                # we need to check this is not larger than our actual number of rows.
+                cell_styles=self._style_cells(
+                    offset,
+                    min(
+                        self._searched_manager.get_num_rows(force=True) or 0,
```

Comment:
can we pull out `self._searched_manager.get_num_rows(force=True)` to line 783? since its also used above
just: `total_rows = self._searched_manager.get_num_rows(force=True)`

Reply:
i just made the change